projects/github-actions/test-results-to-slack/src/index.js

```diff
@@ -20,8 +21,8 @@ const { getMessage, sendMessage } = require( './slack' );
 		return;
 	}
 
-	const channel = getInput( 'slack_channel' );
-	if ( ! channel ) {
+	const defaultChannel = getInput( 'slack_channel' );
+	if ( ! defaultChannel ) {
```

Comment:
We can probably put the function call right into a conditional. Not sure about it tho

Reply:
I'm not sure I understand this one. Can you elaborate a bit?

Reply:
`defaultChannel` is only used in the conditional logic. We can get rid of that variable, do something like this:

```
	if ( ! getInput( 'slack_channel' ) ) {
```